frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableGroup.component.tsx

```diff
@@ -80,41 +81,44 @@ export const TicketsTableGroup = ({ tickets, onEditTicket, onNewTicket, selected
 								: (
-									<Headers>
-										<SortingTableHeader name="id" disableSorting>
-											<FormattedMessage id="ticketTable.column.header.id" defaultMessage="#id" />
-										</SortingTableHeader>
-										<SortingTableHeader name={BaseProperties.TITLE}>
-											<FormattedMessage id="ticketTable.column.header.title" defaultMessage="title" />
-										</SortingTableHeader>
-										<SortingTableHeader name="modelName">
-											<FormattedMessage id="ticketTable.column.header.federationContainer" defaultMessage="federation / container" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`properties.${BaseProperties.CREATED_AT}`}>
-											<FormattedMessage id="ticketTable.column.header.createdAt" defaultMessage="created at" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`properties.${IssueProperties.ASSIGNEES}`}> 
-											<FormattedMessage id="ticketTable.column.header.assignees" defaultMessage="assignees" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`properties.${BaseProperties.OWNER}`}>
-											<FormattedMessage id="ticketTable.column.header.owner" defaultMessage="owner" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`properties.${IssueProperties.DUE_DATE}`}>
-											<FormattedMessage id="ticketTable.column.header.dueDate" defaultMessage="due date" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`properties.${IssueProperties.PRIORITY}`}>
-											<FormattedMessage id="ticketTable.column.header.priority" defaultMessage="priority" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`properties.${BaseProperties.STATUS}`}>
-											<FormattedMessage id="ticketTable.column.header.status" defaultMessage="status" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`modules.safetibase.${SafetibaseProperties.LEVEL_OF_RISK}`}>
-											<FormattedMessage id="ticketTable.column.header.levelOfRisk" defaultMessage="level of risk" />
-										</SortingTableHeader>
-										<SortingTableHeader name={`modules.safetibase.${SafetibaseProperties.TREATMENT_STATUS}`}>
-											<FormattedMessage id="ticketTable.column.header.treatmentStatus" defaultMessage="treatment status" />
-										</SortingTableHeader>
-									</Headers>
+									<>
+										<Headers>
+											<SortingTableHeader name="id" disableSorting>
+												<FormattedMessage id="ticketTable.column.header.id" defaultMessage="#id" />
+											</SortingTableHeader>
+											<SortingTableHeader name={BaseProperties.TITLE}>
+												<FormattedMessage id="ticketTable.column.header.title" defaultMessage="title" />
+											</SortingTableHeader>
+											<SortingTableHeader name="modelName">
+												<FormattedMessage id="ticketTable.column.header.federationContainer" defaultMessage="federation / container" />
```

Comment:
Would there be any drawbacks to using the new COLUMN_NAME_TO_SETTINGS_LABEL from ticketsTable helpers in place of these formatted messages?